assets/css/base.scss

```diff
@@ -419,12 +419,13 @@ th {
 
 .flag__red {
   background-color: rgb(255, 192, 203);
+  border-radius: 1rem;
   font-size: 0.9rem;
   font-weight: bold;
-  border-radius: 1rem;
-  padding: 0.45rem 0.65rem;
   margin: 0.25rem 0.75rem;
+  padding: 0.45rem 0.65rem;
```

Comment:
More a curiosity than an issue if you don't mind explaining (for my own edification): Why did you rearrange these properties? 🤓

Reply:
@tatianamac  Sorry again for delay! Day job got out of hand! I simply reorder them alphabetically out of habit for better readability, but happy to change them back in preference to what you may already have in place throughout CSS.

Reply:
No sorries. Understood; carry on as you are. I was mostly curious.